ruby-gem/app/services/api_maker/validation_errors_generator_service.rb

```diff
@@ -51,6 +51,9 @@ def inspect_model(model, path)
   def attribute_type(model, attribute_name)
     if model.attribute_names.include?(attribute_name.to_s)
       :attribute
+    elsif model.class.const_defined?(:ADDITIONAL_ATTRIBUTES_FOR_VALIDATION_ERRORS) &&
+        model.class.const_get(:ADDITIONAL_ATTRIBUTES_FOR_VALIDATION_ERRORS).include?(attribute_name)
```

Comment:
Couldn't this work without the horrible indentation?
```suggestion
    elsif model.class.const_get(:ADDITIONAL_ATTRIBUTES_FOR_VALIDATION_ERRORS)&.include?(attribute_name)
```

Reply:
Sadly no :( it will raise a name error exception